apps/services/user-profile/src/app/user-profile/userProfile.controller.ts

```diff
@@ -76,9 +86,16 @@ export class UserProfileController {
     nationalId: string,
     @CurrentUser()
     user: User,
+    @Query('actorRequest') actorRequest?: boolean,
```

Comment:
Are you still going to use actor request to this endpoint?

Reply:
No, good point. Will fix.